test/test_evaluate/test_docstring.py

```diff
@@ -1,13 +1,20 @@
 """
 Testing of docstring related issues and especially ``jedi.docstrings``.
+
+CommandLine:
+    # Switch to the right directory
+    cd $(python -c "import jedi; from os.path import dirname, abspath; print(dirname(dirname(abspath(jedi.__file__))))")
+    # Run the doctests in this module
+    tox -e py test/test_evaluate/test_docstring.py
+
```

Comment:
Helpful command line to start running tests quicker.

Reply:
I agree with all your comments in this review except for the first part of this one (although I'll agree the cd is a bit cumbersome). Even though it might be easy to lookup a tox tutorial and figure out how to run tests and then determine how to transfer the commands learned in the tutorial to your own project, I think its immensely helpful to have a copy-pastable command line in a python source file that tells you how to run it. 

I just spent some time reworking some of this PR into a new one. I don't use tox on a regular basis, so I forgot what the exact command was to run the tests. The fact that I came back here, found this note, and didn't have to context switch and wonder "how is this done again?", is a working example for why  having a docstring in your test file (rather than something you have to go searching for in the docs) that tells you how to run it is a very worthwhile .

Reply:
I disagree. This would mean that I would add such a string to every single test module. Generally I would also do testing with pytest and not tox, it's usually faster. just saying ;-)

Many people in the python community are familiar with pytest. It's not an obscure tool. I know that it might take some time to get used to it. I know that it takes some time to read certain documentation. However I think that's just how writing software works. It takes time. Doing things quickly mostly ends up catastrophic (checkout the first versions of Jedi).

Reply:
On my personal projects, I typically do keep a string like that in every module. Usually its just a byproduct of me typing it once, I copy and paste it into the file I'm currently editing, and then it just lives there. I've found that it makes it easier to bring new people onto a project if they have a quick starting point to get the thing they want to test running. Of course py.test can do this, but I historically worked with junior coders, and py.test's syntax requires lots of typing and is not the most intuitive. I've also got an awesome `<leader>a` vim keybinding that copies the current line in my editor into the nearest terminal and executes it, so perhaps my opinion is just a function of my workflow.

Regardless, its your codebase and I respect your design choices. I've got a new PR up in #963 which shouldn't have any of this chaff in it.

Reply:
What is complicated in `pytest <path>`? I cannot really imagine a simpler syntax.

Reply:
I think it would be nice if you could specify a module name (given you've done something like `pip install -e .` without needing the path, similar to python's `-m` flag.  I also feel like `pytest path_to_module::funcname`  and `pytest path_to_module::classname::funcname`is less intuitive than something like `pytest <path_to_module> <funcname>` and `pytest <path_to_module> <classname>.<funcname>`.  Its nit-picky to be sure, but that's my feeling on it. On the other hand, it is nice that the `::` means that a single test node is a single string (with no spaces).

Reply:
I've never even used that. I just use filenames + `-k` which gives me substring matching and which is exactly what i want.